### ISSUE TITLE
project not working after update to blaze 2.2.1  (ok with 2.2.0)

### ISSUE THREAD

**Issue by deeeed** [meteor/blaze#230]
Just tried to update to latest 2.2.1, but I can an error from a custom package using SimpleSchema.

It seems the error comes from adding "custom" property to my schema with:
```
autoform: {
            type: "select-buttons",
            options: "allowed",
            aCustomProp: false,
        }
```
![image](https://cloud.githubusercontent.com/assets/4800095/21793798/85e6216a-d72f-11e6-9423-b8ad1c15c64b.png)

Unfortunately, I don't have time to investigate more. I forced @2.2.0 for now.

**Comment by mitar** [comment]
Duplicate of #228. It has been fixed in master.

**Comment by jykae** [comment]
@mitar this is fix possibly coming to next release? we probably stay in previous release then.

**Comment by mitar** [comment]
Released as 2.3.0.
